compiler/control/OMROptions.cpp

```diff
@@ -450,7 +450,8 @@ TR::OptionTable OMR::Options::_jitOptions[] = {
    {"disableOptTransformations=",         "O{regex}\tlist of optimizer transformations to disable",
                                           TR::Options::setRegex, offsetof(OMR::Options, _disabledOptTransformations), 0, "P"},
    {"disableOSR",                         "O\tdisable support for on-stack replacement", SET_OPTION_BIT(TR_DisableOSR), "F"},
-   {"disableOSRCallSiteRemat",              "O\tdisable use of the call stack remat table in on-stack replacement", SET_OPTION_BIT(TR_DisableOSRCallSiteRemat), "F"},
+   {"disableOSRCallSiteRemat",            "O\tdisable use of the call stack remat table in on-stack replacement", SET_OPTION_BIT(TR_DisableOSRCallSiteRemat), "F"},
+   {"disableOSRLocalRemat",               "O\tdisable use of remat when inserting guards for on-stack replacement", SET_OPTION_BIT(TR_DisableOSRLocalRemat), "F"},
```

Comment:
Should this entry go after disableOSRExceptionEdgeRemoval to keep the alphabetical ordering ?